js/appInit.ts

```diff
@@ -36,13 +36,21 @@ function configureBindings() {
     }
 }
 
+function registerTableView() {
+    window["TableView"] = TableView;
+}
+
+function registerActionBlock() {
+    window["ActionBlock"] = ActionBlock;
+}
+
 function bootstrap(localConfiguration?: Partial<AppConfig>) {
     overrideConfiguration(localConfiguration || {});
 
     // tslint:disable:no-string-literal
     window["ko"] = ko;
-    window["TableView"] = TableView;
-    window["ActionBlock"] = ActionBlock;
+    registerTableView();
```

Comment:
Now it is time to check if global `TableView` used in production. Maybe it here just for testing purposes.